### PR TITLE
DP-7307 & DP-7483 organization image links

### PR DESCRIPTION
**Pull request by isaacchansky**
## Description

Create new Image Link molecule with block variant and Image Link List organism for use on the service page 'offered by' section within the sidebar, or as a stacked row item on the Organization Elected Official page.

## Related Issue / Ticket

- [DP-7307](https://jira.state.ma.us/browse/DP-7307)
- [DP-7483](https://jira.state.ma.us/browse/DP-7483)

## Steps to Test

1. View invision file https://projects.invisionapp.com/share/VYF2N5JX2#/screens/270254806
2. Confirm that the service page 'Offered By' section looks as expected: https://mayflower.digital.mass.gov/?p=pages-service 
3. View invision file https://projects.invisionapp.com/share/VYF2N5JX2#/screens
4. Confirm that the organization elected official page 'Our Organizations' section looks as expected

## Screenshots
You can see these two organisms in action here:
http://www.isaacchansky.com/mayflower/?p=pages-organization-elected-official
and here:
http://www.isaacchansky.com/mayflower/?p=pages-service

## Additional Notes:

Anything else to add?

#### Impacted Areas in Application

* new `molecules/image-link`
* new `organisms/image-link-list`
* change `templates/content-types/services` to use image-link-list instead of link-list
* add to `pages/organization-elected-official` to include image-link-list for 'our organizations' section

#### @TODO

*

### CODE REVIEW

File: styleguide/source/_patterns/03-organisms/by-author/logo-link-list.twig [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <div class="ma__link-list__description">
+      {% set richText = logoLinkList.description %}
+      {% include "@organisms/by-author/rich-text.twig" %}
+    </div>
+  {% endif %}
+
+  <div class="ma__link-list__container">
+      <ul class="ma__link-list__items ma__link-list__items--no-bullets">
+        {% block logoListBlock %}
+        {% for logoLink in logoLinkList.links %}
```
Comment (tnathanjames) [inline styleguide/source/_patterns/03-organisms/by-author/logo-link-list.twig]: Missing indent here.

File: styleguide/source/_patterns/02-molecules/logo-link.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "logoLink": {
+    "text": "Link to another page",
+    "href": "#",
+    "info": "Find out more about this thing on our blah page",
+    "image": {
+        "src":"../../assets/images/placeholder/250x250.png",
```
Comment (tnathanjames) [inline styleguide/source/_patterns/02-molecules/logo-link.json]: Indent 2 spaces instead of 4.

**Review by tnathanjames — CHANGES_REQUESTED** [review]
A couple quick indention corrections.

File: styleguide/source/_patterns/04-templates/01-content-types/services.json [hunk 3]
```diff
@@ -126,12 +126,19 @@
...
       },
       "stacked": true,
       "links" : [{
+        "href":"#",
+        "text":"Link to organization page with logo",
+        "image": {
+          "src": "../../assets/images/placeholder/250x250.png",
+          "alt": "logo"
+        }
+      },{
```
Comment (clair0917) [inline styleguide/source/_patterns/04-templates/01-content-types/services.json:141]: add a carriage return

File: styleguide/source/assets/scss/03-organisms/_link-list.scss [hunk 4]
```diff
@@ -44,6 +44,8 @@
     }
   }
 
+
+
```
Comment (clair0917) [inline styleguide/source/assets/scss/03-organisms/_link-list.scss:48]: extra line

**Review by clair0917 — APPROVED** [review]
looks good to me

**Review by tnathanjames — APPROVED** [review]

**Pull request merged**